### ISSUE TITLE
console logging p5.Vector

### ISSUE THREAD

**Issue by shiffman** [processing/p5.js-web-editor#469]
Related to #468, but I think a different problem so filing a separate issue. Trying to `print()` or `console.log()` a `p5.Vector` object results in the following message: `Converting circular structure to JSON`. We should see: `{x: 100, y: 100, z: 0}`.

```javascript
function setup() {
  let v = createVector(100, 100);
  print(v);
}
```

**Comment by himanshuc3** [comment]
![screen shot 2018-02-20 at 10 28 34 am](https://user-images.githubusercontent.com/19166045/36408043-d72c2d10-1628-11e8-853c-3efa9bcfa288.png)
Console in Firefox shows a JSHINT warning, the expected console message and Cycle object value error message.
Does this error has something to do with **createVector**. If it does, is this an issue for p5.js library instead of editor.

**Comment by catarak** [comment]
this is an error with the console component i think—it doesn't know how to handle class structures which hold references to themselves and thus are cyclical. this is handled in chrome developer tools by having open and close arrows.

**Comment by catarak** [comment]
this is now fixed thanks to #656

**Comment by shiffman** [comment]
Woohoo! I am constantly demonstrating the beauty of console logging `p5.Vector` now! ❤️
